content/azure.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Microsoft Azure
+---
+
+# Microsoft Azure
+
+The `azure` CPI can be used with [Microsoft Azure](https://azure.microsoft.com/).
+
+ * Release: [cloudfoundry-incubator/bosh-azure-cpi-release](https://github.com/cloudfoundry-incubator/bosh-azure-cpi-release)
+ * Issues: [GitHub Issues](https://github.com/cloudfoundry-incubator/bosh-azure-cpi-release/issues)
+ * Slack: [cloudfoundry#bosh-azure-cpi](https://cloudfoundry.slack.com/messages/bosh-azure-cpi)
+
+
+## Concepts
+
+The following table maps BOSH concepts to their Azure-native equivalents.
+
+| BOSH              | Microsoft Azure |
+| ----------------- | --------------- |
+| Availability Zone | [Availability Zone](https://docs.microsoft.com/en-us/azure/availability-zones/az-overview) |
+| Virtual Machine   | [Virtual Machine](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/sizes) |
+| Network Subnet    | [Virtual Network Subnet](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-overview) |
+| Virtual IP        | [Public IP](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-ip-addresses-overview-arm#public-ip-addresses) |
+| Persistent Disk   | [Disk Storage](https://azure.microsoft.com/en-us/services/storage/disks/) and [Managed Disks](https://azure.microsoft.com/en-us/services/managed-disks/) |
+| Disk Snapshot     | [Managed Disk Snapshot](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/managed-disks-overview#managed-disk-snapshots) |
+| Stemcell          |  |
```

Comment:
Is the stemcell link intentionally missing?

Reply:
Was lazy and hadn't finished filling a couple out. I'm not super familiar with some IaaSes, so will send separate PRs asking for individual CPI teams to finish+review in those cases.